apps/routes/lib/parser.ex

```diff
@@ -19,10 +19,19 @@ defmodule Routes.Parser do
       direction_destinations:
         direction_attrs(attributes["direction_destinations"], parse_route_patterns(relationships)),
       description: parse_gtfs_desc(attributes["description"]),
-      fare_class: parse_gtfs_fare_class(attributes["fare_class"])
+      fare_class: parse_gtfs_fare_class(attributes["fare_class"]),
+      line: parse_line(relationships)
     }
   end
 
+  defp parse_line(%{"line" => [head | _] = _lines}) do
+    # TODO figure out how to make this not an array on parsing
+    # Testing shows this as always a single object from the API that gets parsed into an array for some reason
```

Comment:
Plenty of similar functions in our parsers, [example here](https://github.com/mbta/dotcom/blob/5f2f723b3e028c5e4988e0da16324e3f017350f2/apps/schedules/lib/parser.ex#L42). It's typical.